src/vuestic-theme/vuestic-components/va-slider/VaSlider.vue

```diff
@@ -0,0 +1,582 @@
+    <div
+      class="va-slider__container d-flex align--center"
+      ref="elem"
+    >
+      <div
+        class="va-slider__container__track"
+        :style="{ backgroundColor: color }"/>
+      <template v-if="pins">
+        <div
+          v-for="(pin, key) in pinsCol"
+          :key="key"
+          class="va-slider__container__mark"
+          :class="{ 'va-slider__container__mark--active': isRange ? pin * step > val[0] && pin * step < val[1] : pin * step < val}"
+          :style="{ left: pin * step + '%' }"
+        />
+      </template>
+      <template v-if="isRange">
+        <div
+          ref="process"
+          class="va-slider__container__track va-slider__container__track--active"
+          :style="processStyles"
+          @mousedown="moveStart($event, 0)"/>
+        <div
+          ref="dot0"
+          class="va-slider__container__handler"
+          :style="dotStyles[0]"
+          @mousedown="moveStart($event, 0)"
+        >
+          <div
+            v-if="valueVisible"
+            class="va-slider__container__handler-value title"
+          >
+            {{ val[0] }}
+          </div>
+        </div>
+        <div
+          ref="dot1"
+          class="va-slider__container__handler"
+          :style="dotStyles[1]"
+          @mousedown="moveStart($event, 1)"
+        >
+          <div
+            v-if="valueVisible"
+            class="va-slider__container__handler-value title">
+            {{ val[1] }}
+          </div>
+        </div>
+      </template>
+      <template v-else>
+        <div
+          ref="process"
+          class="va-slider__container__track va-slider__container__track--active"
+          :style="processStyles"
+          @mousedown="moveStart($event, 0)"/>
+        <div
+          ref="dot"
+          class="va-slider__container__handler"
+          :style="dotStyles"
+          @mousedown="moveStart"
+        >
+          <div
+            v-if="valueVisible"
+            class="va-slider__container__handler-value title">
+            {{ labelValue ? labelValue : val }}
+          </div>
+        </div>
+      </template>
+    </div>
+    <span
+      v-if="iconRight"
+      class="va-slider__invert-label title">
+      <va-icon :icon="iconRight" :color="color" :size="16"/>
+    </span>
+    <span
+      v-if="invertLabel"
+      class="va-slider__invert-label title">
+      {{ label }}
+    </span>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'va-slider',
+  props: {
+    range: {
+      type: Boolean,
+    },
+    value: {
+      type: Number | Array,
+    },
+    labelValue: {
+      type: String
+    },
+    valueVisible: {
+      type: Boolean,
+    },
+    min: {
+      type: Number,
+      default: 0
+    },
+    max: {
+      type: Number,
+      default: 100
+    },
+    step: {
+      type: Number,
+      default: 1
+    },
+    color: {
+      type: String,
+      default: 'success'
+    },
+    label: {
+      type: String,
+    },
+    invertLabel: {
```

Comment:
for my opinion, it must be inverseLabel
or inversedLabel and it will be the text of label, that placed in the end. In this way we give ability to use two label  not just choose the place